### PR TITLE
Add DEIMOS lines

### PR DESCRIPTION
**Pull request by profxj**
Update line lists for DEIMOS arc exposures.

Tested somewhat extensively with the 830G and 1200G
gratings.

Added new (weaker) lines into NIST as well.

### CODE REVIEW

File: arclines/data/NIST/XeI_vacuum.ascii [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Retreived from NIST by JXP on 14 Apr 2016
-# Vacuum, 4000-12000Ang
-# Min Rel Int = 1000.
+# Retreived from NIST by JXP on 03 May 2018
+# Vacuum, 8300-12000Ang
```
Comment (rcooke-ast) [inline arclines/data/NIST/XeI_vacuum.ascii]: But the list goes down to 8200A - Also, I'm guessing there are no lines blueward of this, which is why we are not taking from 4000 anymore?

File: arclines/plots.py [hunk 2]
```diff
@@ -75,19 +79,26 @@ def chk_line_list(wave):
 
     # Unknowns
     U_lines = src_dict['U_lines']
-    clrs = ['red', 'orange']  # Used, not used
+    clrs = ['red', 'orange', 'green']  # Used, not used, ??
```
Comment (rcooke-ast) [inline arclines/plots.py]: Two comments - one about color-blindedness (red/green) and the other about the color correspondence (which seems counter intuitive, to me at least). Should we have the following:
used = blue
not used = red
?? = yellow/black

**Review by rcooke-ast — APPROVED** [review]
Thanks for the update. Any report on the success? Do both algorithms (i.e. general and semi-brute) succeed?

**Comment by profxj** [comment]
Both algorithms do succeed and both DEIMOS gratings
have been added to the holy test.

File: arclines/data/NIST/XeI_vacuum.ascii (hunk 1 above)
Comment (profxj) [reply arclines/data/NIST/XeI_vacuum.ascii]: Edited to 8200.   And, yes, I don't know of anyone
vetting XeI down below 8000A.

File: arclines/plots.py (hunk 2 above)
Comment (profxj) [reply arclines/plots.py]: I have to admit my own color scheming confuses me here.

Am going to keep it for now but you can bet it will change again..

**Comment by profxj** [comment]
merging.

**Pull request merged**